gui/src/components/preboarding/admin-user.vue

```diff
@@ -277,9 +282,37 @@ export default class CortxAdminUser extends Vue {
       invalidMessage: "",
       createUserInProgress: false,
       passwordTooltipMessage,
-      usernameTooltipMessage
+      usernameTooltipMessage,
+      systemStatus: true
     };
   }
+  public async mounted() {
+    await this.getSyetmStatus();
+  }
+  public async getSyetmStatus() {
+    this.$store.dispatch(
+      "systemConfig/showLoader",
+      "checking service status..."
+    );
+    try {
+      const dbName = { key: "consul"};

```

Comment:
Is it consul or consult ? I assume there is no spelling mistake, also create a new enum for the same.

Reply:
its consul